src/scripts/main.js

```diff
@@ -358,3 +358,27 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+
+const dashboard = document.querySelector('table.dashboard');
```

Comment:
Consider checking if the `dashboard` element is found before proceeding. If `dashboard` is `null`, attempting to append rows will cause an error. You can add a conditional check to ensure `dashboard` is not `null` before the loop.